### PR TITLE
Checks the language for extensions translations #3149

### LINKED ISSUE #3149: Panel Translation error
**Describe the bug**  
There is a problem in the panel with the translation (not all languages).
If you set (por example) spanish it throw an error
`Undefined array key "es"` 

**To Reproduce**  
1. Download a plainkit project.
2. Create config/config.php
3. Enable languages with `'languages' => true`
4. Create a language file `site/languages/es.php`

```php
<?php

return [
    'default' => true,
    'code' => 'es',
    'direction' => 'ltr',
    'locale' => 'es_MX',
    'name' => 'Español',
    'url' => '/',
    'translations' => [
        'lang' => 'Español',
    ]
];
```

5. Go to <url>/panel
6. An empty page will show

Note: If the `translations` key is empty the no error will show.

**Expected behavior**  
The panel page (installation) should be shown

**Screenshots**  

**Kirby Version**  
The version affected is `3.5.2` in the `3.5.1` and below worked as expected.

**Console output**  
image

**Desktop (please complete the following information):**  
 - OS:  macOS 11.2.1
 - Browser: Chrome, Safari
 - PHP: 7.3, 7.4, 8.0

**Additional context**  
There is no problem with the templates, this issue only affect to the panel.

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

Fixes the issue with checks the language for extensions translations.

## Related issues

- Fixes #3149 

## Ready?

- [x] Added unit tests for fixed bug/feature
- [x] Added in-code documentation (if needed)
- [x] CI passes (runs automatically when the PR is created or run `composer ci` locally)
  Running locally requires PHPUnit, PHP-CS-Fixer, Psalm, PHPCPD and PHPMD.

- [x] Checked whether the PR needs documentation, if needed added to the [release docs checklist](https://github.com/getkirby/getkirby.com/pulls)

### CODE REVIEW

**Review by bastianallgeier — COMMENTED** [review]
Could you add a test case for the notice error?

File: tests/Cms/App/AppTranslationsTest.php [hunk 1]
```diff
@@ -114,6 +114,12 @@ public function testTranslationFromCurrentLanguage()
                     'translations' => [
                         'button' => 'Knopf'
                     ]
+                ],
+                [
+                    'code'         => 'en',
+                    'translations' => [
+                        'hello' => 'Hello'
+                    ]
```
Comment (afbora) [inline tests/Cms/App/AppTranslationsTest.php:122]: @bastianallgeier This is for notice error. You can try with rollback `src/Cms/AppTranslations.php` update in this PR.

Comment (afbora) [reply tests/Cms/App/AppTranslationsTest.php:122]: Tests for: `EN` translations exists but `EN` language translations not exists.

Comment (bastianallgeier) [reply tests/Cms/App/AppTranslationsTest.php:122]: Ah, nice! I missed that.

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**